<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #f0f0f0,
        stop:1 #e0e0e0
    );
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>236</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: qlineargradient(
        spread:pad, 
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #ffffff,
        stop:1 #eaeaea
    );
    border: 1px solid #cccccc;
    border-radius: 6px;
    padding: 8px;
    font-size: 22px;
    font-weight: bold;
    color: #000000;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_c">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>72</y>
      <width>58</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #cccccc;
    color: #000000;
    font-weight: bold;
    border: 1px solid #d0d0d0; 
    border-radius: 10px;
    font-size: 15px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #b3b3b3;
border: 1px solid #b0b0b0;
}

QPushButton:pressed {
    background-color: #999999;
}
</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plusMinus">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>72</y>
      <width>58</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #cccccc;
    color: #000000;
    font-weight: bold;
    border: 1px solid #d0d0d0; 
    border-radius: 10px;
    font-size: 15px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #b3b3b3;
border: 1px solid #b0b0b0;
}

QPushButton:pressed {
    background-color: #999999;
}
</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_perc">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>72</y>
      <width>58</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #ff9500;
    color: #ffffff;
    font-weight: bold;
    border: 1px solid #d0d0d0; 
    border-radius: 10px;
    font-size: 15px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #e08900;
border: 1px solid #b0b0b0;
}

QPushButton:pressed {
    background-color: #cc7a00;
}
</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>72</y>
      <width>58</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #ff9500;
    color: #ffffff;
    font-weight: bold;
    border: 1px solid #d0d0d0; 
    border-radius: 10px;
    font-size: 15px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #e08900;
border: 1px solid #b0b0b0;
}

QPushButton:pressed {
    background-color: #cc7a00;
}
</string>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>131</y>
      <width>58</width>
      <height>58</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f2f2f2;   /* light gray */
    color: #000000;
    border: 1px solid #d0d0d0;
    border-radius: 8px;
    font-size: 18px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #e0e0e0;
border: 1px solid #b0b0b0;
}

QPushButton:pressed {
    background-color: #cfcfcf;
}
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>131</y>
      <width>58</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f2f2f2;   /* light gray */
    color: #000000;
    border: 1px solid #d0d0d0;
    border-radius: 8px;
    font-size: 18px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #e0e0e0;
border: 1px solid #b0b0b0;
}

QPushButton:pressed {
    background-color: #cfcfcf;
}
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>131</y>
      <width>58</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f2f2f2;   /* light gray */
    color: #000000;
    border: 1px solid #d0d0d0;
    border-radius: 8px;
    font-size: 18px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #e0e0e0;
border: 1px solid #b0b0b0;
}

QPushButton:pressed {
    background-color: #cfcfcf;
}
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mult">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>131</y>
      <width>58</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #ff9500;
    color: #ffffff;
    font-weight: bold;
    border: 1px solid #d0d0d0; 
    border-radius: 10px;
    font-size: 15px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #e08900;
border: 1px solid #b0b0b0;
}

QPushButton:pressed {
    background-color: #cc7a00;
}
</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>190</y>
      <width>58</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f2f2f2;   /* light gray */
    color: #000000;
    border: 1px solid #d0d0d0;
    border-radius: 8px;
    font-size: 18px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #e0e0e0;
border: 1px solid #b0b0b0;
}

QPushButton:pressed {
    background-color: #cfcfcf;
}
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>58</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f2f2f2;   /* light gray */
    color: #000000;
    border: 1px solid #d0d0d0; 
    border: 1px solid #d0d0d0;
    border-radius: 8px;
    font-size: 18px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #e0e0e0;
border: 1px solid #b0b0b0;
}

QPushButton:pressed {
    background-color: #cfcfcf;
}
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>58</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f2f2f2;   /* light gray */
    color: #000000;
    border: 1px solid #d0d0d0; 
    border: 1px solid #d0d0d0;
    border-radius: 8px;
    font-size: 18px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #e0e0e0;
border: 1px solid #b0b0b0;
}

QPushButton:pressed {
    background-color: #cfcfcf;
}
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sub">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>58</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #ff9500;
    color: #ffffff;
    font-weight: bold;
    border: 1px solid #d0d0d0;
    border-radius: 10px;
    font-size: 15px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #e08900;
border: 1px solid #b0b0b0;
}

QPushButton:pressed {
    background-color: #cc7a00;
}
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>250</y>
      <width>58</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f2f2f2;   /* light gray */
    color: #000000;
    border: 1px solid #d0d0d0;
    border-radius: 8px;
    font-size: 18px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #e0e0e0;
border: 1px solid #b0b0b0;
}

QPushButton:pressed {
    background-color: #cfcfcf;
}
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>58</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f2f2f2;   /* light gray */
    color: #000000;
    border: 1px solid #d0d0d0;
    border-radius: 8px;
    font-size: 18px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #e0e0e0;
border: 1px solid #b0b0b0;
}

QPushButton:pressed {
    background-color: #cfcfcf;
}
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>58</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f2f2f2;   /* light gray */
    color: #000000;
    border: 1px solid #d0d0d0;
    border-radius: 8px;
    font-size: 18px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #e0e0e0;
border: 1px solid #b0b0b0;
}

QPushButton:pressed {
    background-color: #cfcfcf;
}
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>58</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #ff9500;
    color: #ffffff;
    font-weight: bold;
    border: 1px solid #d0d0d0;
    border-radius: 10px;
    font-size: 15px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #e08900;
border: 1px solid #b0b0b0;
}

QPushButton:pressed {
    background-color: #cc7a00;
}
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>310</y>
      <width>58</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f2f2f2;   /* light gray */
    color: #000000;
    border: 1px solid #d0d0d0;
    border-radius: 8px;
    font-size: 18px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #e0e0e0;
border: 1px solid #b0b0b0;
}

QPushButton:pressed {
    background-color: #cfcfcf;
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dec">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>58</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f2f2f2;   /* light gray */
    color: #000000;
    border: 1px solid #d0d0d0;
    border-radius: 8px;
    font-size: 18px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #e0e0e0;
border: 1px solid #b0b0b0;
}

QPushButton:pressed {
    background-color: #cfcfcf;
}
</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>310</y>
      <width>120</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #ff8a2b;
    color: #ffffff;
    font-weight: bold;
    border: 1px solid #d0d0d0;
    border-radius: 10px;
    font-size: 15px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #e08900;
    border: 1px solid #b0b0b0;
}

QPushButton:pressed {
    background-color: #cc7a00;
}
</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
